Formula/wine.rb

```diff
@@ -38,10 +38,13 @@ def require_universal_deps?
     MacOS.prefer_64_bit?
   end
 
+  option "without-win64", "Build without 64-bit support"
+
   # Wine will build both the Mac and the X11 driver by default, and you can switch
   # between them. But if you really want to build without X11, you can.
   depends_on :x11 => :recommended
   depends_on "pkg-config" => :build
+  depends_on :xcode => ["8.0", :build] if build.with? "win64"
```

Comment:
Whoops, just noticed another problem here: if we build with `win64` by default this will unfortunately mean that by default this fails on Yosemite and below.

This can probably be fixed by changing the above to:
```ruby
if MacOS.version >= :el_capitan
  option "without-win64", "Build without 64-bit support"
  depends_on :xcode => ["8.0", :build] if build.with? "win64"
end
```
so it refuses to try and build Win64 on older versions of macOS (with older versions of Xcode). Sorry I didn't spot this sooner.